Sources/MapboxCoreNavigation/NavigationServiceDelegate.swift

```diff
@@ -29,6 +29,18 @@ public protocol NavigationServiceDelegate: AnyObject, UnimplementedLogging {
      */
     func navigationService(_ service: NavigationService, shouldRerouteFrom location: CLLocation) -> Bool
     
+    /**
+     Asks permission to proceed with found proactive reroute and apply it is main route.
```

Comment:
apply it **as** main route?

Reply:
Yes, this is the terminology we use to identify 'main', 'alternative' routes...